index.html

```diff
@@ -0,0 +1,126 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+  <link rel="stylesheet" href="styles.css">
+</head>
+
+<body>
+<h1>Hello World!</h1>
+  <header id="header">
+    <div class="logo">
+      <img id="header-img" src="https://cdn.freecodecamp.org/testable-projects-fcc/images/product-landing-page-logo.png"
+        alt="original trombones logo">
+    </div>
+
+    <nav id="nav-bar">
+      <ul>
+        <li><a class="nav-link" href="#features">Features</a></li>
+        <li><a class="nav-link" href="#how-it-works">How It Works</a></li>
+        <li><a class="nav-link" href="#pricing">Pricing</a></li>
+      </ul>
+    </nav>
+  </header>
+  <div class="heading">
+    <h2>Handcrafted, home-made masterpieces</h2>
+  </div>
+  <form id="form" action="https://www.freecodecamp.com/email-submit">
+    <input name="email" id="email" type="email" placeholder="Enter your email address" required="">
+    <input id="submit" type="submit" value="Get Started" class="btn">
+  </form>
```

Comment:
Nice job so far, 👏 , however, Please ensure that you use `<section>` tag instead of `<div>` using the `<section>` tag instead of a generic `<div>` tag brings more semantic meaning to your HTML code, which improves SEO, accessibility and maintainability. It also helps create a clearer documentation structure, making it easier for developers to understand and modify the content layout. 🙂